openpype/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -45,7 +45,10 @@ class VrayRenderPluginInfo():
 @attr.s
 class RedshiftRenderPluginInfo():
     SceneFile = attr.ib(default=None)
-    Version = attr.ib(default=None)
+    # "1" was selected as the default RS version as it's the default
+    #  version in RS deadline plugin.
```

Comment:
```suggestion
    # Use "1" as the default Redshift version just because it
    # default fallback version in Deadline's Redshift plugin
    # if no version was specified
```